File: domain-service/credit-application/src/main/java/at/deloittedigital/core_banking/credit/datasource/model/Account.java
package at.deloittedigital.core_banking.credit.datasource.model;

import java.time.LocalDate;
import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Account {
    private String accountId;
    private LocalDate createdDate;
    private List<Transaction> transactions;
}

File: data-service/src/entity/Account.ts
import { ObjectType, Field } from "type-graphql";
import { Transaction } from "./Transaction";

@ObjectType()
export class Account {
  @Field()
  accountId: string;

  @Field()
  frequency: string;

  @Field()
  createDate: string;

  @Field(() => [Transaction])
  transactions: Transaction[];
}
